app/factories/bulkrax/object_factory.rb

```diff
@@ -243,18 +243,14 @@ def update_file_set(attrs)
     def handle_remote_file(remote_file:, actor:, update: false)
```

Comment:
with these changes, update is an unused variable. Do we still need it?

Reply:
yes, probably. I walked the code path and I _think_ calling create_content all the time instead of sometimes calling update_content is the right call. You are attaching an unknowable file (remote source could change it) if you are downloading it from a url, so it makes sense to me that it has to be treated as new. I only have a confidence level of 72.5% on this though

Reply:
updated